addons/im_livechat/models/res_partner.py

```diff
@@ -43,6 +44,8 @@ def _search_for_channel_invite_to_store(self, store: Store, channel):
                     # sudo: res.users.settings - operator can access other operators expertises
                     "livechat_expertise": partner.user_ids.sudo().livechat_expertise_ids.mapped("name"),
```

Comment:
I wonder, shouldn't the sudo be placed before user_ids? As if the current partner cannot read users it will just be an empty recordset there.

To be tested with a user with limited access.